usaspending_api/api_docs/api_documentation/download/README.md

```diff
@@ -0,0 +1,40 @@
+# Download
+
+The following endpoints are involved with generating files that reflect the site's underlying data. 
+
+## Award Data Archive
+
+On a monthly basis, the website pre-generates a series of commonly used files based on the agency, fiscal year, and award type. These can be found at [Award Data Archive](https://beta.usaspending.gov/#/bulk_download/award_data_archive), or, if you prefer, you can also use the API's [List Downloads Endpoint](https://github.com/fedspendingtransparency/data-act-documentation/blob/master/usaspending/api-documentation/download/List%20Downloads.md).
+
+## Generating Download Files
+
+**BEFORE USING THESE ENDPOINTS, MAKE SURE YOU'VE DETERMINED WHAT YOU NEED IS NOT IN THE AWARD DATA ARCHIVE**
+
+There are several downloadable endpoints, all with different features/constraints. 
+
+### Row Constraint Downloads
+
+These downloads have a row constraint on them, meaning they have a hard limit to the number of records to include (currently that limit is `500,000` rows). The main benefit of using these endpoints, however, is that they allow various filters that are not supported by the Year Constraint Downloads.
+
+For downloading transactions, please use [Advanced_Search_Transaction_Download](https://github.com/fedspendingtransparency/data-act-documentation/blob/master/usaspending/api-documentation/download/Advanced_Search_Transaction_Download.md).
+
+For downloading awards, please use [Advanced_Search_Award_Download](https://github.com/fedspendingtransparency/data-act-documentation/blob/master/usaspending/api-documentation/download/Advanced_Search_Award_Download.md).
```

Comment:
Both of these links as well as `Custom_Award_Data_Download` and `Status Endpoint` below are 404

Reply:
noted